src/ClientData/CaptureDataTest.cpp

```diff
@@ -123,4 +132,49 @@ TEST_F(CaptureDataTest, UpdateScopeStatsIsCorrect) {
                    GetStats(kDurationsForSecondId, kSecondVariance));
 }
 
+TEST_F(CaptureDataTest, VarianceIsCorrectForLongDurations) {
+  for (TimerInfo timer : kTimerInfos) {
+    timer.set_end(timer.end() + kLargeInteger);
+    capture_data_.UpdateScopeStats(timer);
+  }
+
+  capture_data_.UpdateScopeStats(kTimerInfoWithInvalidScopeId);
+
+  ORBIT_LOG("VARIANCE %f", capture_data_.GetScopeStatsOrDefault(kFirstId).variance_ns());
+  EXPECT_NEAR(capture_data_.GetScopeStatsOrDefault(kFirstId).variance_ns(), kFirstVariance, 1.0);
+  EXPECT_NEAR(capture_data_.GetScopeStatsOrDefault(kSecondId).variance_ns(), kSecondVariance, 1.0);
+}
+
+TEST_F(CaptureDataTest, VarianceIsCorrectOnScimitarDataset) {
+  // The file first line of the file contains the expected variance. The rest of the lines store
+  // durations one per line. The last line is empty.
+  std::filesystem::path path = orbit_test::GetTestdataDir() / "scimitar_variance_and_durations.csv";
+  const ErrorMessageOr<std::string> file_content_or_error = orbit_base::ReadFileToString(path);
+  EXPECT_TRUE(file_content_or_error.has_value());
+  const std::string& file_content = file_content_or_error.value();
+  const std::vector<absl::string_view> tokens = absl::StrSplit(file_content, '\n');
+
+  double expected_variance;
+  EXPECT_TRUE(absl::SimpleAtod(*tokens.begin(), &expected_variance));
+
+  std::vector<TimerInfo> timers;
+  std::transform(std::begin(tokens) + 1, std::end(tokens) - 1, std::back_inserter(timers),
+                 [](const absl::string_view& line) {
+                   uint64_t duration = std::stoull(std::string(std::begin(line), std::end(line)));
```

Comment:
Is there a reason why line is an `absl::string_view` and not a `std::string_view`? You could the `std::string_view` to create the string without the begin/end: `std::string(line)`.

Reply:
No good reason at all. Thanks!